advisories/a103220.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Technical Advisory 103220
+advisory: A-103220
+summary: Inserting rows into a multi-column-family table with COPY can corrupt the table, making future reads fail with internal errors.
+toc: true
+affected_versions: v23.1.0-alpha.1 to v23.1.0
+advisory_date: 2023-05-16
+docs_area: releases
+---
+Publication date: {{ page.advisory_date | date: "%B %e, %Y" }}
+
+## Description
+
+In CockroachDB v23.1.0 and its testing versions, inserting rows into a multi-[column-family](../v23.1/column-families.html) table with [`COPY`](../v23.1/copy-from.html) can corrupt the table. Future reads on the corrupted table can result in internal errors or silent data corruption. If the table is empty before the `COPY` executes, the table must be dropped and re-[copied](../v23.1/copy-from.html) to be encoded correctly. If data was copied into a table with existing rows, the data in those rows may be irrecoverable.
+
+## Statement
+
+This is resolved in CockroachDB by PR [103323](https://github.com/cockroachdb/cockroach/pull/103323).
+
+This fix has been applied to maintenance releases of CockroachDB [v23.1.1](../releases/v23.1.html#v23-1-1) and later.
+
+This public issue is tracked by [#103220](https://github.com/cockroachdb/cockroach/issues/103220).
+
+## Mitigation
+
+Users running CockroachDB v23.1.0 can avoid this issue by setting the [session variable](../v23.1/set-vars.html) `vectorize` to `off`, or setting `copy_fast_path_enabled` to `off` in any session that executes the `COPY` command.
+
+Users running CockroachDB v23.1.0 are encouraged to upgrade to v23.1.1 or a later version as soon as possible.
+
+## Impact
```

Comment:
It seems relevant and helpful to add an extra paragraph here like:

The v23.1.0 binary was withdrawn hours after its release, and prior to the formal announcement of this major release, so impacts to production workloads are not likely.